supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,13 @@
+# Column: CommitmentDiscountType
```

Comment:
Would love it if we can capture some of the discussion topics. If its all captured in teh commitmentdiscountcategory.md file, just link to that file here (relative link should work in supporting content).

Reply:
Ok, relative link added